EventsExpress/ClientApp/src/components/event/users-view.js

```diff
@@ -0,0 +1,66 @@
+﻿import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { getAttitudeClassName } from './attitude';
+import CustomAvatar from '../avatar/custom-avatar';
+import Tooltip from '@material-ui/core/Tooltip';
+import Zoom from '@material-ui/core/Zoom';
+import * as moment from 'moment';
+import './users-view.css';
+
+export default class UserView extends Component{
+    constructor() {
+        super();
+    }
+
+
+    getAge = birthday => {
+        let today = new Date();
+        var date = moment(today);
+        var birthDate = moment(birthday);
+        let age = date.diff(birthDate, 'years');
+
+        if (age >= 100) {
+            age = "---";
+        }
+
+        return age;
+    }
+
+    render() {
+        const { user } = this.props;
+        const att = user.attitude; 
+
+        return( <div>
+            <div className={"d-flex align-items-center border-bottom w-100 " + getAttitudeClassName(att)} >
+                <div className="flex-grow-1" >
+                    <Link to={'/user/' + user.id} className="btn-custom">
+                        <div className="d-flex align-items-center">
+                            <CustomAvatar size="little" photoUrl={user.photoUrl} name={user.username} />
+                            <div>
+                                <h5>{user.username}</h5>
+                                {'Age: ' + this.getAge(user.birthday)}
+                            </div>
+                            {user.attitude === 0 &&
```

Comment:
This code is duplicated with the one in the user-card-info. Consider having a single component for that.